src/Simulation/Simulators/Microsoft.Quantum.Simulators.csproj

```diff
@@ -47,4 +47,11 @@
 
   </ItemGroup>
 
+  <ItemGroup>
```

Comment:
Is this item group still needed? I believe there should be no SparseSimulatorDeprecated folder?